src/components/card-text-only/card-text-only.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class CardTextOnly extends React.Component {
+  render() {
+    const { props } = this;
+    return (
+      <div className="col col--12 mb12">
```

Comment:
I think that the parent component should determine this element's position in the context of a layout. Looks like that responsibility falls under CardContainer.

It may be that we could even eliminate this wrapper div.